applications/ConvectionDiffusionApplication/custom_elements/symbolic_d_convection_diffusion_explicit.cpp

```diff
@@ -0,0 +1,1019 @@
+            rhs[3]=-DN_DX_3_0*crhs92 - DN_DX_3_1*crhs93 - DN_DX_3_2*crhs94 + crhs10*crhs223 - 0.5854102*crhs101 + crhs105 - 0.5854102*crhs111 + crhs115 - 0.5854102*crhs121 + crhs125 + crhs137 + crhs138*(crhs129 + crhs131 + crhs212 + 0.34270510226404*phi[3] - 0.34270510226404*phi_old[3]) + crhs151 + crhs156*crhs222 + crhs158*crhs223 + crhs16*crhs223 + crhs160*crhs224 + crhs162*crhs225 + crhs164 - crhs170*crhs222 - crhs172*crhs223 - crhs174*crhs224 - crhs175*crhs225 - crhs176*crhs222 - crhs177*crhs224 + crhs178 - crhs179*crhs225 + 0.5854102*crhs179 - crhs181*crhs222 - crhs182*crhs223 - crhs183*crhs224 - crhs184*crhs225 + crhs208 + crhs218*tau[0] + crhs218*tau[1] + crhs218*tau[2] + crhs218*tau[3] + crhs219*tau[0] + crhs219*tau[1] + crhs219*tau[2] + crhs219*tau[3] + crhs220*tau[0] + crhs220*tau[1] + crhs220*tau[2] + crhs220*tau[3] + crhs221*tau[3] - crhs221 + crhs222*crhs79 + crhs222*crhs81 - crhs223*crhs64 + crhs224*crhs83 + crhs224*crhs86 + crhs225*crhs88 + crhs225*crhs90 + crhs70 - 0.5854102*crhs71 + crhs75 + 0.5854102*crhs76 + crhs84 + crhs87 - 0.5854102*crhs88 - 0.5854102*crhs90 + 0.19999999899376*f[2] + 0.40000000301872*f[3];
+
+
+    // All the weights of the gauss points are the same so we multiply by volume/n_nodes
+    const double local_size = 4;
+    noalias(rRightHandSideVector) = rhs * rData.volume/local_size;
+
+    KRATOS_CATCH("");
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+template <>
+void SymbolicDConvectionDiffusionExplicit<2,3>::CalculateOrthogonalSubgridScaleRHSInternal(
+    BoundedVector<double, 3>& rRightHandSideVector,
+    const ProcessInfo& rCurrentProcessInfo)
+{
+    KRATOS_TRY;
+
+    // Element data
+    ElementData rData;
+    this->InitializeEulerianElement(rData,rCurrentProcessInfo);
+
+    // Compute tau
+    this->CalculateTau(rData);
+
+    // Retrieve element data
+    const auto& k = rData.diffusivity;
```

Comment:
k usually stands for the conductivity. Diffusivity is usually denoted with alpha. I strongly suggest to make this consistent as it will avoid future misunderstandings (we have suffered enough with the viscosity ...).

Reply:
Done. I was not aware of this.